layout/PrimaryNav/PrimaryNavLink/__tests__/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+import PrimaryNavLink from '../index'
+
+describe('PrimaryNavLink', function() {
+  var primaryNavLink;
```

Comment:
No but they actually have the same scoping in this context, so it doesn't really matter which one we use. Will change to `let` for consistency though.